packages/react-dom/src/client/ToStringValue.js

```diff
@@ -18,8 +18,8 @@ export opaque type ToStringValue =
 // Flow does not allow string concatenation of most non-string types. To work
 // around this limitation, we use an opaque type that can only be obtained by
 // passing the value through getToStringValue first.
-export function toString(value: ToStringValue): string {
-  return '' + (value: any);
+export function toString(value: ?ToStringValue): string {
+  return value == null ? '' : '' + (value: any);
```

Comment:
Why this change? Did we check every `toString` callsite to ensure it doesn't break them?

Reply:
There were two places in postMount where it could stringify null or undefined. Otherwise this can stay the same. Updating now.

Reply:
My priority here is that the old code path stays as similar as it can be. So that we can merge this without worrying.

Reply:
Totally. I've done this in e4f68fee7